src/NovelRT/DotNet/RuntimeService.cpp

```diff
@@ -73,7 +73,7 @@ namespace NovelRT::DotNet {
       if (result != 0)
       {
         _logger.logError("Failed to initialize the runtime: {}", result);
-        throw std::runtime_error("Failed to initialize the runtime");
+        throw Exceptions::InitialisationFailureException("Failed to initialize the runtime",result);
```

Comment:
US spelling.

Also, do we want to consider maybe making the exception message more unified?